.workleap.rules.yaml

```diff
@@ -114,4 +74,73 @@ rules:
     severity: warn
     given: $.servers[*].url
     then:
-      function: falsy
\ No newline at end of file
+      function: falsy
+
+  must-not-use-basic-auth:
+    description: "Contract must not use basic auth"
+    recommended: true
+    severity: warn
+    given: $.components.securitySchemes.*
+    then:
+      field: scheme
+      function: pattern
+      functionOptions:
+        notMatch: '^basic$'
+
+  # Backend to Backend specific rule
```

Comment:
No need to specify if it's application to B2B or B4F, the respective files that do opt-in is where the truth